pom.xml

```diff
@@ -64,6 +65,21 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
```

Comment:
This is wrong -- I already fixed it earlier... what happened? Maybe need to rebase/merge from `main`?

I can fix it after merge, not a big deal, more worried about other changes getting reverted.

Reply:
Actually you should just remove these entries: Jackson YAML, jdk8 deps are already included around line 200/220.

Reply:
removed the jackson-datatype-jdk8 and jackson-dataformat-yaml from the dependencyManagement section and  confirmed they are both below.